common/dna_data/wc_elf_dna.txt

```diff
@@ -0,0 +1,484 @@
+			gene_bs_nose_ridge_angle={ "nose_ridge_angle_neg" 181 "nose_ridge_angle_neg" 33 }
+			gene_bs_nose_ridge_width={ "nose_ridge_width_neg" 3 "nose_ridge_width_neg" 3 }
+			gene_bs_nose_size={ "nose_size_neg" 5 "nose_size_neg" 44 }
+			gene_bs_nose_tip_angle={ "nose_tip_angle_pos" 5 "nose_tip_angle_neg" 50 }
+			gene_bs_nose_tip_forward={ "nose_tip_forward_pos" 9 "nose_tip_forward_neg" 9 }
+			gene_bs_nose_tip_width={ "nose_tip_width_pos" 121 "nose_tip_width_pos" 39 }
+			face_detail_cheek_def={ "cheek_def_01" 116 "cheek_def_01" 148 }
+			face_detail_cheek_fat={ "cheek_fat_01_neg" 75 "cheek_fat_01_neg" 141 }
+			face_detail_chin_cleft={ "chin_cleft" 0 "chin_dimple" 13 }
+			face_detail_chin_def={ "chin_def" 153 "chin_def" 21 }
+			face_detail_eye_lower_lid_def={ "eye_lower_lid_def" 222 "eye_lower_lid_def" 245 }
+			face_detail_eye_socket={ "eye_socket_03" 255 "eye_socket_01" 67 }
+			face_detail_nasolabial={ "nasolabial_03" 255 "nasolabial_02" 129 }
+			face_detail_nose_ridge_def={ "nose_ridge_def_pos" 91 "nose_ridge_def_neg" 111 }
+			face_detail_nose_tip_def={ "nose_tip_def" 255 "nose_tip_def" 212 }
+			face_detail_temple_def={ "temple_def" 246 "temple_def" 246 }
+			expression_brow_wrinkles={ "brow_wrinkles_01" 255 "brow_wrinkles_01" 40 }
+			expression_eye_wrinkles={ "eye_wrinkles_02" 218 "eye_wrinkles_02" 218 }
+			expression_forehead_wrinkles={ "forehead_wrinkles_02" 0 "forehead_wrinkles_02" 132 }
+			expression_other={ "cheek_wrinkles_both_01" 137 "cheek_wrinkles_left_01" 127 }
+			complexion={ "complexion_1" 243 "complexion_1" 243 }
+			gene_height={ "full_height" 130 "normal_height" 169 }
+			gene_bs_body_type={ "body_fat_head_fat_full" 113 "body_fat_head_fat_full" 113 }
+			gene_bs_body_shape={ "body_shape_average_clothed" 102 "body_shape_rectangle_half" 6 }
+			gene_bs_bust={ "bust_clothes" 71 "bust_shape_3_half" 71 }
+			gene_age={ "old_beauty_1" 0 "old_4" 152 }
+			gene_eyebrows_shape={ "no_eyebrows" 75 "no_eyebrows" 75 }
+			gene_eyebrows_fullness={ "no_eyebrows" 89 "no_eyebrows" 104 }
+			gene_body_hair={ "body_hair_sparse" 0 "body_hair_sparse" 0 }
+			hairstyles={ "western_hairstyles" 161 "high_elven_hairstyles" 181 }
+			beards={ "steppe_beards" 127 "high_elven_beards" 237 }
+			eye_accessory={ "normal_eyes" 51 "normal_eyes" 51 }
+			teeth_accessory={ "normal_teeth" 0 "normal_teeth" 0 }
+			eyelashes_accessory={ "normal_eyelashes" 247 "normal_eyelashes" 247 }
+			gene_bs_ear_lenght={ "erect_ear_lenght" 204 "erect_ear_lenght" 246 }
+			special_eyebrows={ "high_elven_eyebrows" 255 "high_elven_eyebrows" 25 }
+			special_eyes={ "magic_eyes" 37 "magic_eyes" 37 }
+			gene_bs_mouth_tusks={ "mouth_tusks_pos" 0 "mouth_tusks_pos" 0 }
+			static_tusks={ "no_static_tusks" 0 "no_static_tusks" 0 }
+			gene_bs_ear_tip={ "ear_tip_pos" 0 "ear_tip_pos" 0 }
+            gene_skin_value={ "skin_value_neg" 0 "skin_value_neg" 0 }
+		}
+	}
+	enabled=yes
+}
+
+sylvanas_windrunner_dna = { # by ercarp
```

Comment:
She does look pretty odd. Not sure what happened there, but it could also just be the animation she's using since those accentuate certain facial features.

Reply:
Looks fine on my end, but I was able to replicate the issue using a different idle animation.

<details><summary>Click to expand</summary>

![image](https://user-images.githubusercontent.com/78575425/114316713-56ea2e00-9b0d-11eb-9eb3-e272e672a9a5.png)

</details>